packages/opds-web-client/src/components/Book.tsx

```diff
@@ -144,10 +144,11 @@ export default class Book<P extends BookProps> extends React.Component<P, {}> {
 
     if (this.isOpenAccess()) {
       if (this.props.epubReaderUrlTemplate) {
+        let index = 0;
         for (const link of this.props.book.openAccessLinks) {
           if (link.type === "application/epub+zip") {
             links.push(
-              <span key={link.url}>
+              <span key={`${link.url}-${index}`}>
```

Comment:
Added these to remove the console warnings about react flatten children having the same key in a render loop.